### DISCUSSION TITLE
added ripgrep-git to scoop

### DISCUSSION THREAD

**Discussion by ingenarel** [BurntSushi/ripgrep#2905] [General]
i'm stuck using windows, and i really miss the aur git packages, so i made a [scoop bucket](https://github.com/ingenarel/ingenarel-scoop-bucket) and i'm currently adding git packages to scoop.

the [ripgrep-git package](https://github.com/ingenarel/ingenarel-scoop-bucket/blob/master/bucket/ripgrep-git.json) is auto maintained, i made a python script to check the latest commit and update the hash if needed, and it runs every hour using github actions. (using more could make github actions finish up my monthly 2k minutes)

i'm mostly packaging stuff for fun, but hopefully this will help the community in some way.

**Comment by BurntSushi** [comment]
For things like this, it would be a lot more helpful if you could say when folks might want to use this. What is the difference between this and other Windows installation methods shown in the README?

**Reply by ingenarel** [reply]
it's like an aur -git package. for those who don't know, lemme explain it simply.

it downloads the latest git repo, and builds ripgrep

the package version is the latest commit hash of the ripgrep repo. 

i autoupdate the commit hash using github actions, and it checks for the latest commit hash every hour.

it's like the bleeding edge package system like the aur offers. but for people who're using windows instead of arch.

hopefully this answers the question.
